### PR TITLE
contribute decently fast yggdrasil address generator in C

### PR DESCRIPTION
**Pull request by fifteenthcommotion**
I think the yggdrasil-brute-simple address generator is now mature.
https://github.com/fifteenthcommotion/yggdrasil-brute-simple

### CODE REVIEW

File: contrib/yggdrasil-brute-simple/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*
```
Comment (Mikaela) [inline contrib/yggdrasil-brute-simple/.gitignore]: I wonder if you intend this directory to be a submodule instead? https://git-scm.com/book/en/v2/Git-Tools-Submodules

File: contrib/yggdrasil-brute-simple/LICENSE [hunk 2]
```diff
@@ -0,0 +1 @@
+As public domain as possible. Don't blame me if your computer explodes.
```
Comment (Mikaela) [inline contrib/yggdrasil-brute-simple/LICENSE]: I think CC0 might be more legal way to say this. https://creativecommons.org/choose/zero/

Comment (fifteenthcommotion) [reply contrib/yggdrasil-brute-simple/LICENSE]: Apparently CC0 has some problems with explicitly not waiving patent rights.
https://opensource.stackexchange.com/questions/133/how-could-using-code-released-under-cc0-infringe-on-the-authors-patents

I'll probably use the Unlicense as it seems to be fairly short boilerplate for releasing code into the public domain and disclaiming warranty.

I expect to rarely (hopefully never) have to modify this code and I would rather not complicate the process of `git clone`-ing.

**Comment by fifteenthcommotion** [comment]
Licensing is kind of a mess so I decided to add the CC0 dedication as well. There are now half as many licensing lines as there are lines of code.

**Comment by neilalexander** [comment]
Thanks very much for contributing this!

**Pull request merged**